README.md

```diff
@@ -12,7 +12,9 @@ You can find the changes for each version in the [change log](https://github.com
 
 ## Samples and Documentation
 
-[We provide a full suite of sample applications and documentation on GitHub](https://github.com/azure-samples?query=active-directory) to help you get started with learning the Azure Identity system. This includes tutorials for native clients such as Windows, Windows Phone, iOS, OSX, Android, and Linux. We also provide full walkthroughs for authentication flows such as OAuth2, OpenID Connect, Graph API, and other awesome features. 
+[We provide a full suite of sample applications and documentation on GitHub](https://github.com/azure-samples?query=active-directory) to help you get started with learning the Azure Identity system. This includes tutorials for native clients such as Windows, Windows Phone, iOS, OSX, Android, and Linux. We also provide full walkthroughs for authentication flows such as OAuth2, OpenID Connect, Graph API, and other awesome features.
+ 
+Instructions for setting up an Azure AD application are here, [Azure Active Directory authentication in Azure Portal](https://docs.microsoft.com/en-us/azure/app-service-mobile/app-service-mobile-how-to-configure-active-directory-authentication).
```

Comment:
I think saying that additional information is available in the samples would be good enough. I thought providing the link here would tie the two systems together more clearly. Thus, it doesn't matter where you start investigating Azure AD, you can get to the documentation you need to get started building an app.